Modelica/Constants.mo

```diff
@@ -53,8 +53,11 @@ package Constants
   final constant SI.Permeability mu_0 = 4*pi*1.00000000055e-7 "Magnetic constant";
   final constant Real epsilon_0(final unit="F/m") = 1/(mu_0*c*c)
     "Electric constant";
-  final constant NonSI.Temperature_degC T_zero=-273.15
-    "Absolute zero temperature";
+  final constant SI.Temperature T_zero_K = -273.15
```

Comment:
```suggestion
  final constant SI.Temperature T_zero_K = -0
```
Absolute zero is just zero, but I don't think anyone needs this.

Reply:
Of course!